.mvn/modernizer/violations.xml

```diff
@@ -43,6 +43,37 @@
         <comment>Prefer Math.toIntExact(long)</comment>
     </violation>
 
+    <violation>
+        <name>com/google/common/collect/ImmutableList$Builder."&lt;init&gt;":()V</name>
+        <version>1.8</version>
+        <comment>Use builder() static factory method instead</comment>
+    </violation>
+    <violation>
+        <name>com/google/common/collect/ImmutableMap$Builder."&lt;init&gt;":()V</name>
+        <version>1.8</version>
+        <comment>Use builder() static factory method instead</comment>
+    </violation>
+    <violation>
+        <name>com/google/common/collect/ImmutableMultimap$Builder."&lt;init&gt;":()V</name>
+        <version>1.8</version>
+        <comment>Use builder() static factory method instead</comment>
+    </violation>
+    <violation>
+        <name>com/google/common/collect/ImmutableMultiset$Builder."&lt;init&gt;":()V</name>
+        <version>1.8</version>
+        <comment>Use builder() static factory method instead</comment>
+    </violation>
+    <violation>
+        <name>com/google/common/collect/ImmutableSet$Builder."&lt;init&gt;":()V</name>
+        <version>1.8</version>
+        <comment>Use builder() static factory method instead</comment>
+    </violation>
+    <violation>
+        <name>com/google/common/collect/ImmutableTable$Builder."&lt;init&gt;":()V</name>
+        <version>1.8</version>
+        <comment>Use builder() static factory method instead</comment>
+    </violation>
+
```

Comment:
I suppose these are the ones the most likely to be encountered "in the wild", but should we also pre-emptively prohibit things like `ImmutableMultiset`, `ImmutableBiMap`, `ImmutableSortedSet`, `ImmutableSortedMap` and a couple of others?

Reply:
> `ImmutableMultiset`, `ImmutableBiMap`, `ImmutableSortedSet`, `ImmutableSortedMap` a

ImmutableMultiset is already here (did you mean something else?)
adding the others

Reply:
> did you mean something else?

Probably just missed it